src/components/Button/Button.stories.tsx

```diff
@@ -243,3 +258,100 @@ Loading.args = {
   loading: true,
   disabled: true,
 };
+
+export const SecondarySuccess = Template.bind({});
```

Comment:
I tossed all the extra combos in at the end so people can see that they are available but they have lower visibility than the preferred combos above.

Reply:
it'd probably be easiest to reference these in a Grid-layout story (like the current "All Variants" one or something)